proposals/nnnn-safe-shared-contiguous-storage.md

```diff
@@ -177,6 +201,14 @@ extension UnsafeMutableRawBufferPointer: ContiguousStorage<UInt8> {
 }
 ```
 
+**TODO**: What is the `@_unsafeNonescapableResult` annotation? Would `Slice<UnsafeBufferPointer<UInt8>>` need it?
```

Comment:
This is for when there is no safe binding to which a lifetime can be attached. It turns out that this includes any pointer binding, because the abi assumes these will be passed by register, it's baked in. `Slice<UBP<UInt8>>` won't require the annotation explicitly, since it uses it's base's conformance in a generic context. I'm not clear on what this means when specialization occurs.

Reply:
Should we mention it in the proposal, or should the lifetime annotations proposal talk about it?

Reply:
I brought it up with the compiler team, and I think they will add it to the annotations proposal.